rust/lon/src/git.rs

```diff
@@ -191,6 +192,119 @@ pub fn get_last_modified(url: &str, rev: &str) -> Result<u64> {
         .context("Failed to parse last modified timestamp.")
 }
 
+/// List the commits between two revisions
+pub fn diff_history(
```

Comment:
I find the term `diff` here to be suboptimal because it usually means something different in git. Let's call this `rev-list` as that's basically what we're doing here.